src/plugins/data_view_field_editor/public/types.ts

```diff
@@ -22,13 +22,13 @@ import { FormatEditorServiceSetup, FormatEditorServiceStart } from './service';
 import { DeleteFieldProviderProps } from './components';
 
 export interface PluginSetup {
-  fieldFormatEditors: FormatEditorServiceSetup['fieldFormatEditors'];
+  fieldFormatEditors: FormatEditorServiceSetup;
 }
 
 export interface PluginStart {
   openEditor(options: OpenFieldEditorOptions): () => void;
   openDeleteModal(options: OpenFieldDeleteModalOptions): () => void;
-  fieldFormatEditors: FormatEditorServiceStart['fieldFormatEditors'];
+  fieldFormatEditors: FormatEditorServiceStart;
```

Comment:
I'm not sure if something got convoluted, or if the original code was wrong somehow. It's strange that `FormatEditorServiceStart` doesn't have a `fieldFormatEditors` property in this PR.